### DISCUSSION TITLE
Wix v4 - console output

### DISCUSSION THREAD

**Discussion by Steve0212a** [wixtoolset/issues#7965] [Questions]
Is it possible to output to the console from a custom bootstrapper?  I would like my installer support a custom UI or just console output depending on the command line switches.  

I put a console.writeline in the constructor of my BootstrapperApplication, but it did not appear on the console.  

Is this possible?  If so, how?

thanks

**Comment by robmen** [comment]
If you figure out a good solution to creating a Windows executable that can output to the console it was launched from like you suggest, let me know. Only solutions I've ever seen were crazy hacks (that won't work from installation).

**Reply by Steve0212a** [reply]
I was afraid you would say that.  I too have not found a good way to do that...
